gbpservice/nfp/scripts/mode_shift_enterprise.sh

```diff
@@ -97,7 +111,7 @@ function configure_visibility_user_data {
 # $1 is the Visibility VM's IP address
     CUR_DIR=$PWD
     visibility_vm_ip=$1
-    configurator_ip=
+    configurator_ip=`neutron port-show pt_configuratorVM_instance -f value -c fixed_ips | cut -d'"' -f8`
```

Comment:
The name of configurator instance should be taken from result file of advanced mode installation as this can change. Lets plan to fix any such things in next commit.